File: Java/Maths/Find largest and smallest number from array - input values.java
import java.util.Scanner;
import java.util.Arrays;
class largestSmallestNumberFromArray_input {
    public static void main(String args[]){
        Scanner input = new Scanner(System.in);
        System.out.print("How many number should have the array: ");
        int size = input.nextInt();
        int[] array = new int[size];
        int largest = 0, smallest = 0;
        for (int i = 0; i < array.length; i++) {
            System.out.print("Enter the number-"+(i+1)+": ");
            array[i] = input.nextInt();
            if (i == 0) {
                largest = smallest = array[i];
            } else{
                if (array[i] > largest) {
                    largest = array[i];
                }
                if (array[i] < smallest) {
                    smallest = array[i];
                }
            }
        }
        System.out.println("\nYour array is: " + Arrays.toString(array));
        System.out.println("Largest number is: " + largest);
        System.out.println("Smallest number is: " + smallest);
    }
}

File: Python/Maths/Find largest and smallest number from array - input values.py
size = int(input("How many number should have the array: "))
array = [None] * size
largest = smallest = 0
for i in range(0, len(array)):
    array[i] = int(input("Enter the number-"+str(i+1)+": "))
    if i == 0:
        largest = smallest = array[i]
    else:
        if array[i] > largest:
            largest = array[i]

        if array[i] < smallest:
            smallest = array[i]

print("\nYour array is:", array)
print("Largest number is:", largest)
print("Smallest number is:", smallest)